README.rst

```diff
@@ -615,3 +619,99 @@ Stats
 
 Stats from python-zyte-api_ are exposed as Scrapy stats with the
 ``scrapy-zyte-api`` prefix.
+
+
+Request fingerprinting
+======================
+
+The request fingerprinter class of this plugin ensures that Scrapy 2.7 and
+later generate unique `request fingerprints
+<https://docs.scrapy.org/en/latest/topics/request-response.html#request-fingerprints>`_
+for Zyte API requests based on some of their parameters.
+
+For example, a request for ``browserHtml`` and a request for ``screenshot`` are
+considered different requests. Similarly, requests with different ``actions``
+are also considered different requests.
+
+Zyte API parameters that affect request fingerprinting
+------------------------------------------------------
+
+The request fingerprinter class of this plugin generates request fingerprints
+for Zyte API requests based on the following Zyte API parameters:
+
+-   ``url`` (`canonicalized <https://w3lib.readthedocs.io/en/latest/w3lib.html#w3lib.url.canonicalize_url>`_)
+
+    URL canonicalization keeps the URL fragment, if any, if
```

Comment:
```suggestion
    URL canonicalization keeps any fragments, in cases where
```

I needed to re-read this a couple of times since the first `if` creates some slight confusion.